scripts/record-milestone.sh

```
@@ -0,0 +1,122 @@
+    echo "Error: Missing required parameters"
+    echo "Usage: $0 <milestone_type> <success_rate> <current_date> <total_tests> <passed_tests> <datetime>"
+    exit 1
+fi
+
+# Generate milestone content based on type
+case "$MILESTONE_TYPE" in
+    "completion")
+        EMOJI="🎉"
+        TITLE="E2Eテスト100%達成"
+        DESCRIPTION="### 🎯 完全な成功達成!" \
+            $'\n\n' \
```

Shellcheck (0.10.0)

[error] This is interpreted as a command name containing a linefeed. Double check syntax.

(SC2289)